### DISCUSSION TITLE
Add Tooltips for beginners

### DISCUSSION THREAD

**Discussion by kongomongo** [teslamate-org/teslamate#721] [Ideas]
**What would you like to be added**:
Tooltips to explain certain non-selfexplanatory things

**Why is this needed**:
For clarification

**Additional context**:
For example the Vampire Drain dashboard:
Theres a Standby column in percent: what does it mean? Whats TR loss mean? Other columns say kWh and Consumption and dont really explain the difference. Or the ❄ column. Etc...

Thanks

**Comment by adriankumpf** [comment]
Are there other columns where a tooltip would be useful? Could you please list them?

**Comment by kongomongo** [comment]
Alright, let's do it dashboard by dashboard (please bear with me here..):

**Charges:**
Used =>What does Used mean in the context? Used since last charge?
Efficiency => Efficiency of Charging as reported by ... car?
kW = average power during charging
km/h = km gained per hour while charging
%Start,End = Battery SOC at Start/End of Charging
Driven = ?

**Charging Stats:**
Duration = Total Time spent Charging?
Charge Heatmap = Charging shown by Time of Day?
Charge/Discharge Stats, SOC = Number of times Charged/[Started drive] to/from this battery level

**Drives:**
%Start,End = Battery SOC at Start/End of Driving
Temp = Average/min/max inside/outside/? temperature during drive
Speed = Average Speed during drive?
max Power = maximum battery/motor Power output during drive.
❄ = Was battery below optimal temperatur during drive?
kWh = amount of energy consumed during drive (how is it calculated?)
Consumption = average real/rated(?) power consumption in kWh/100km during drive
Efficiency = this one is challenging....

**Efficiency:**
min. distance per drive = does not change the net/gross consumption values, only for tables?

Efficiency = how/what is calculated?
km = total km driven at that temperature
Speed = average speed ?

**Locations:**
Adresses, Updated at = last visited at?

**Projected Range:**
Charged = Show charging sessions?
Also that interval setting = needs some explanation...

**Trip:**
first 3 panels showing km and km/h dont really explain anything..
used = power consumed
net/gross = what kind of values are they? how/why are they different from other dashboards?
km, start/end%, consumption = see above

**Vampire Drain:**
min idle time = ??? 
Period = period of standby/sleep/nondrive time??
Standby % = ???
SOC = battery power/soc change within that period
TR loss = ?
kWh = amount of energy/kWh lost during period?
Ø-Power = average power consumption/vampire drain during that period
TR Loss / h = ??

**Reply by MarcFinns** [reply]
In the charges dashboard, what is the meaning of the column driven?

**Comment by adriankumpf** [comment]
Excellent, thank you 🙏
